charts/sequencer/values.yaml

```diff
@@ -37,15 +39,15 @@ config:
         - 34fec43c7fcab9aef3b3cf8aba855e41ee69ca3a
     # Note large balances must be strings support templating with the u128 size account balances
     genesisAccounts:
-    - address: 1c0c490f1b5528d8173c5de46d131160e4b2c0c3
-      balance: "333333333333333333"
-    - address: 34fec43c7fcab9aef3b3cf8aba855e41ee69ca3a
-      balance: "333333333333333333"
-    - address: 60709e2d391864b732b4f0f51e387abb76743871
-      balance: "333333333333333333"
-      # NOTE - the following address matches the privKey that funds the sequencer-faucet
-    - address: 00d75b270542084a54fcf0d0f6eab0402982d156
-      balance: "333333333333333333"
+      - address: 1c0c490f1b5528d8173c5de46d131160e4b2c0c3
```

Comment:
just to confirm - does the genesis support both hex and bech32 now?

Reply:
So the chart is indifferent, when you run in dev mode it builds the old genesis that is intended to work with the latest release. When you run in `dev` mode it builds the new genesis which works on this PR and latest after this is merged.

Reply:
Yeah to chime in: this is only for CI. The sequencer genesis itself only supports proper bech32m addresses.